### PR TITLE
Adapt expected values 2

### PR DESCRIPTION
**Pull request by franzhaas**
I run the tests and saw some failing, because the generated code was prettier than the expectations.

I fixed the expectations to match the prettier formating, and configured traivis ci

### CODE REVIEW

**Comment by auvipy** [comment]
@mkwiatkowski

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: python
+python:
+  - "2.6"
+  - "2.7"
+  - "3.5"
```
Comment (mkwiatkowski) [inline .travis.yml]: Pythoscope doesn't support Python 3, so I don't think there's a point in listing python 3.5 in travis configuration.

File: pythoscope/serializer.py [hunk 2]
```diff
@@ -278,8 +278,13 @@ def __init__(self, obj, serialize):
...
 
-    # def __repr__(self):
-    #     return "SequenceObject(%s)" % (self.constructor_format % self.contained_objects)
+#    def __eq__(self, other):
+#        return False
+           
+            
+#    def __repr__(self):
+#        import ipdb; ipdb.set_trace()
+#        return "SequenceObject(%s)" % (self.constructor_format % self.contained_objects)
```
Comment (mkwiatkowski) [inline pythoscope/serializer.py]: Please don't add more dead code. This `__repr__` was probably commented out by accident.

**Comment by auvipy** [comment]
@mkwiatkowski shouldn't it be ported to python 3?

**Comment by mkwiatkowski** [comment]
@auvipy It should, but it isn't at the moment. There's no point in having travis running a constantly failing suite.

**Comment by franzhaas** [comment]
Hi,

well I actually do not mind test results that fail, I only mind those that
are wrong (ie false positives or false negatives).

But if you want I obliously can remove the python3 part...

Best regards,
Franz

2016-08-10 19:23 GMT+02:00 Michal Kwiatkowski <notifications@github.com>:

**Comment by mkwiatkowski** [comment]
@franzhaas I've manually pulled some of your changes into the repo. Thanks!

**Pull request closed without merging**